tensorflow/tools/ci_build/linux/rocm/rocm_py310_pip.sh

```diff
@@ -42,8 +42,20 @@ export N_TEST_JOBS=$(expr ${TF_GPU_COUNT} \* ${TF_TESTS_PER_GPU})
 # Get the default test targets for bazel.
 source tensorflow/tools/ci_build/build_scripts/DEFAULT_TEST_TARGETS.sh
 
+rocm_major_version=`cat /opt/rocm/.info/version | cut -d "." -f 1`
+rocm_minor_version=`cat /opt/rocm/.info/version | cut -d "." -f 2`
+echo $rocm_major_version
```

Comment:
debug lines?

Reply:
Ah forgot to clean that up for Python 310 file

Reply:
Pushed an update